Creates a new command to list a compact profile list

Godoy opened the pull request:
Creates the command `analytics profiles list`. 

Two reasons: 
- The original command `pageviews list` causes a very long response in Rocket Chat because it gets the metadata of all returned urls, including page title, image, description.. and duplicates all infos in response.
- Change the command prefix in a plugin's generic name, allowing expand to other calls to the Google Analytics API. Something like `analytics 12345678 mobile` to get access mobile vs desktop or `analytics 12345678 browser` for most used browsers.

Godoy committed:
Creates a new command to list a compact profile list

mostafazh commented:
First of all thanks for the PR :)

Secondly, I can see that you're doing the same api request and getting the same response, so I don't get why this won't cause the same problems on Rocket Chat?

Godoy commented:
Hi!
Thank you for the excellent plugin!

I did it to keep the return from the old method. I just changed the response: "#{item.id} - #{item.name}". The field "item.url" causes a big response because the RocketChat gets meta-info from the page, like facebook does.

I changed the name so we can create new features (like #6 ) with names that are compatible with google analytics api.

Godoy committed:
Fixes error feedback on 'analytics profiles list'
